be/src/vec/exprs/table_function/vexplode_json_array.cpp

```diff
@@ -59,17 +62,28 @@ void VExplodeJsonArrayTableFunction<DataImpl>::process_row(size_t row_idx) {
 
     StringRef text = _text_column->get_data_at(row_idx);
     if (text.data != nullptr) {
-        rapidjson::Document document;
-        document.Parse(text.data, text.size);
-        if (!document.HasParseError() && document.IsArray() && document.GetArray().Size()) {
-            _cur_size = _parsed_data.set_output(document, document.GetArray().Size());
+        if (WhichDataType(_text_datatype).is_json()) {
```

Comment:
Maybe we should use template?

Reply:
here using a same sign with old rapid , so can not use template, just according data